sdcm/nemesis.py

```diff
@@ -676,6 +676,19 @@ def repair_streaming_exists():
         self.log.debug("Execute a complete repair for target node")
         self.repair_nodetool_repair()
 
+    def disrupt_validate_hh_short_downtime(self):
+        """
+            Validates that hinted handoff mechanism works: there were no drops and errors
+            during short stop of one of the nodes in cluster
+        """
+        self._set_current_disruption("ValidateHintedHandoffShortDowntime")
+        start_time = time.time()
+        self.target_node.stop_scylla()
```

Comment:
Maybe we should add a sleep here to better control the gap time?
Also we can measure the gap time and make sure it's consistent or not al least not greater than X.

We may want to avoid cases where it took to stop/start scylla too long and it causes errors later for hints.